canonical-web-and-design/ubuntu.com
# Add spans to prevent unwanted wrapping on k8s/features

## Done

- Added some markup to stop wrapping issues with the list of code block links.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/features
- Go to the "storage" section and see that "(PersistentVolumeClaims, PersistentVolumes, StorageClasses)" wraps in reasonable ways
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7472
